File: FashionInBerland.java
// Link: http://codeforces.com/problemset/problem/691/A

import java.util.Scanner;

public class FashionInBerland {
	public static void main(String[] args)
	{
		
		int n, temp = 0, button;
		Scanner input = new Scanner(System.in); 
		n = input.nextInt();
		for (int i = 0; i < n; ++i)
		{
			button = input.nextInt();
			if (button ==  1)
				++temp;
		}
		
		if (temp == 1 && n == 1)
		{
			System.out.println("YES");
		}
		
		if (temp == (n-1))
		{
			System.out.println("YES");
		}
		else
		{
			System.out.println("NO");
		}
	}
}

File: FashionInBerland.cpp
/*
Link: http://codeforces.com/problemset/problem/691/A
*/


#include<stdio.h>

int main(int argc, char** agrv)
{
	int n, k, t = 0, y = 0;
	scanf("%d", &n);
	y = n;
	while (y--) {
		scanf("%d", &k);
		if (k == 1)
			++t;
	}

	if (n == 1 && k == 1)
		printf("YES\n");

	else if (n > 1 && t == n - 1)
		printf("YES\n");
	else 
		printf("NO\n");
	return 0;
	
}
